src/api/helpers.ts

```diff
@@ -39,6 +39,7 @@ const NotAuthenticatedCalls = [
   'native/v1/signin',
   'native/v1/validate_email',
   'native/v1/offer',
+  'native/v1/institutional-project-redactor-account',
```

Comment:
elle correspond à quoi cette route ? `refresh_access_token` par exemple c'est clair, mais là je vois pas ce que ça peut faire

Reply:
c'est trop tard mtn, mais pour les prochaines

Reply:
C'est comme `'native/v1/account'` mais pour un rédacteur de projet, mais j'avoue que c'est un peu long

Reply:
Bien noté, mais enfait même `account` c'est pas une action donc c'est pas clair. Ça aurait plus clair à mon sens avec `native/v1/create_account`. Mais ça c'est encore plus trop tard ^^